src/main/java/com/minecolonies/achievements/MineColoniesAchievement.java

```diff
@@ -12,8 +12,7 @@
  *
  * @since 0.2
  */
-public class MineColoniesAchievement extends Achievement
-{
+public class MineColoniesAchievement extends Achievement {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)